source/style-guide/style/toc-labels.txt

```diff
@@ -0,0 +1,113 @@
+
+When you create a TOC, you can generate these labels automatically 
+based on the titles of your pages, or specify labels that differ from 
+the page title.
+
+Page titles and Table of Contents (TOC) labels serve distinct purposes. 
+We should craft them with different considerations.
+
+TOC labels and page titles can match. However, we should define them 
+separately even when they match as a good habit and for added 
+flexibility. This approach ensures that we can easily implement any 
+future changes without affecting the other element. Separate 
+definitions for TOC labels and page titles contribute to organized 
+content management and a seamless user experience.
+
+While both page titles and TOC labels contribute to the overall user 
+experience, page titles focus more on attracting users and SEO, while 
+TOC labels prioritize representing the document's structure and aiding 
+in navigation within the document itself.
+
+Page Titles
+-----------
+
+- Optimize page titles for search engines. They should include relevant 
+  keywords to improve the page's discoverability in search results. To 
+  learn more, see :ref:`seo-guidelines`.
+
+- Create succinct, meaningful, descriptive page titles, and place the 
+  most important words first. Page titles should have a maximum of 70 
+  characters and provide a quick overview of the content. Search 
+  results or browser tabs might truncate extra long titles. Avoid 
+  excessive or irrelevant words (keyword stuffing).
+
+- While we should craft concise titles, include enough information to 
+  confirm to the user that they have landed in the right place. A page 
+  title can provide a bit more detail than a TOC label.
+
+- Ensure that each title and heading is unique. Identical titles, even 
+  between documentation sets, might compete in search results.
+
+- Don't include "MongoDB" in a title unless the page is a product 
+  landing page.
+
+- Aim to attract users and encourage them to click on the link.
+
+- Create page titles that are readable and effective on various 
+  devices, including mobile. This involves considering character limits 
+  and readability on smaller screens.
+
+- Use a consistent grammatical structure, style, and tone across all 
+  page titles to maintain a unified and professional appearance. For 
+  example, use verbs to indicate procedural content.
+
+
+To learn more, see :ref:`titles-and-headings`.
+
+TOC Labels
+----------
+
+- While the automatic labels make it easier for us to 
+  update the TOC as the content evolves, long titles interfere with 
+  efficient navigation. So, we recommend that you specify TOC labels to 
+  enhance the document's accessibility and usability.
+
+- Craft clear and descriptive TOC labels that convey the content of 
+  each section or page accurately. Users should understand the context 
+  from the TOC label and its placement in the overall TOC hierarchy.
+
+- Avoid line-wrapping. Let the placement of the TOC label in the 
```

Comment:
[Question]

Just making sure I'm reading this correctly, this is saying that TOC labels should appear on a single line of text? If so, maybe we can consider framing this as what _to_ do, rather than what _not to_ do:

```suggestion
- Ensure your TOC label fits on a single line of text. Let the placement of the TOC label in the 
```